connector/jobrunner/runner.py

```diff
@@ -160,12 +160,25 @@ def _openerp_now():
     dt = datetime.datetime.utcnow()
     return _datetime_to_epoch(dt)
 
+def _connection_info_for(db_name):
+    connection_info = {'database': db_name}
+    for p in ('host', 'port', 'user', 'password'):
+        cfg = config['db_' + p]
```

Comment:
Why not fallback to `openerp.sql_db.connection_info_for` when there is no key `job_runner_db_host`, so we would benefits from the uri parsing and have less duplication?